forte/data/caster.py

```diff
@@ -78,3 +81,40 @@ def input_pack_type():
     @staticmethod
     def output_pack_type():
         return MultiPack
+
+
+class DataPackBoxer(Caster[MultiPack, DataPack]):
+    """
+    This pass on a single DataPack within the MultiPack.
+    """
+
+    def cast(self, pack: MultiPack, pack_index: int = 0) -> DataPack:
```

Comment:
The `cast` function signature should be consistent, we should not add this additional `pack_index` attribute in this function.

As we've discussed before, we should move this to the configs.